### ISSUE TITLE
Data missing from CSV incidents download

### ISSUE THREAD

**Issue by jim-mcgowan** [redcross/dcsops#321]
When downloading data as a CSV from the **More Incidents...** page, the following fields (columns) aren't populated:

- num_adults
- num_children
- num_families
- assistance_given
- incident_verified
- dispatch_relayed
- responders_identified
- dat_on_scene
- dat_departed_scene

<img width="1440" alt="GNY" src="https://user-images.githubusercontent.com/4147733/125139013-56920c80-e0d5-11eb-938e-f795e4d8614c.png">

**Comment by jim-mcgowan** [comment]
@frankduncan: Now when I attempt to download incident data (via the **Download CSV** link), I get a small CSV file titled **incidents.csv** containing the following:

<img width="404" alt="CSV" src="https://user-images.githubusercontent.com/4147733/127500392-3d907a11-63d0-4fdb-bcbc-4fec9102da0d.png">


This remains urgent as Greater New York is trying to complete end-of-FY reporting.

Thanks!

**Comment by frankduncan** [comment]
It just worked for me, can you try again?

**Comment by jim-mcgowan** [comment]
I just tried downloading GNY data from July 2020 - July 2021 and I get the same problem. Interestingly, it takes 60 seconds from the time I click **Download CSV** to the when the file downloads (I timed it). Also, every time I try, the file overwrites the previous file in my downloads folder (rather than creating a new file).

**Comment by jim-mcgowan** [comment]
@frankduncan: If this is a more involved fix, it can wait until tomorrow. I'll put off GNY for another day.

**Comment by frankduncan** [comment]
Yeah, it's timing out.  If you do only a month, it works.  Something in the code I added to fix this issue (most likely iterating over the timeline for every incident a few times to get those dates) is making it slow down.

So yes, this a more involved fix.

**Comment by jim-mcgowan** [comment]
@frankduncan: OK. Considering they want data back to January 1, 2018, I think we'll wait for the more involved fix (rather than do 43 individual downloads).

**Comment by frankduncan** [comment]
I've pushed a fix that lets me get 6 months of data with no problem.  Maybe it'll work for longer ones as well, but they may have to piece together a few sheets.
